subsys/nrf_security/src/drivers/cracen/cracenpsa/src/ctr_drbg.c

```diff
@@ -158,6 +166,27 @@ psa_status_t cracen_init_random(cracen_prng_context_t *context)
 	return silex_statuscodes_to_psa(sx_err);
 }
 
+psa_status_t cracen_reseed(void)
+{
+	psa_status_t status = PSA_SUCCESS;
+
+	/* DRBG entropy + nonce buffer. */
+	char entropy[CRACEN_PRNG_ENTROPY_SIZE +
+		     CRACEN_PRNG_NONCE_SIZE];
```

Comment:
No need for newline it seems?
```suggestion
	char entropy[CRACEN_PRNG_ENTROPY_SIZE + CRACEN_PRNG_NONCE_SIZE];
```